samtranslator/model/iam.py

```diff
@@ -63,3 +63,31 @@ def dead_letter_queue_policy(cls, action, resource):
                 }]
             }
         }
+
+    @classmethod
+    def sqs_send_message_role_policy(cls, queue_arn, logical_id):
+        document = {
+            'PolicyName': 'SQSPublishPolicy' + logical_id,
```

Comment:
Should I change the naming convention to logical_id + `SQSPublishPolicy`?

Reply:
What is the existing convention?

Reply:
existing convention is FunctionName+'integer number'. But I cannot follow this convention as it might conflict with the explicit `Policies` section.
`logical_id` is `FunctionName+EventName`

Reply:
updated the policy name to <event-logicalid>+<sqspolicy/snspolicy>